src/build-change-graph.js

```diff
@@ -116,15 +118,26 @@ async function buildChangeGraph({
       continue;
     }
 
+    let changedReleasableFiles = await getChangedReleasableFiles({
+      changedFiles: newFiles,
+      packageCwd: _package.cwd,
+      workspacesCwd: workspaceMeta.cwd,
+    });
+
     let dag = buildDAG(workspaceMeta, _package.packageName);
 
     packagesWithChanges[dag.packageName] = {
       changedFiles: newFiles,
+      changedReleasableFiles,
       dag,
     };
   }
 
-  for (let { dag } of Object.values(packagesWithChanges)) {
+  for (let { dag, changedReleasableFiles } of Object.values(packagesWithChanges)) {
```

Comment:
what's `dag`?

Reply:
directed acyclic graph

Reply:
I understand the need to abbreviate that :D